File: data/c/pattern_12/code.c
#include <stdio.h>

int main(void)
{
    int num;
    int x;
    printf("Enter the number of rows and columns for the square: ");
    scanf("%d", &num);

    for (int i = 1; i <= num; i++)
    {
        x = i;
        for (int j = 1; j <= num; j++)
        {
            printf("%d ", x);
            x += num;
        }
        printf("\n");
    }

    return 0;
}

File: data/python/pattern_12/code.py
#MAIN PROGRAM STARTS HERE:
num = int(input('Enter the number of rows and columns for the square: '))

for x in range(0, num):
    i = x + 1
    for y in range(0, num):
        print ('{} '.format(i), end='')
        i += num
    print()